confidential-data-hub/secret/src/bin/secret_cli.rs

```diff
@@ -58,9 +79,18 @@ struct EnvelopeCommand {
 #[derive(Subcommand)]
 enum EnvelopeArgs {
     /// Alibaba KMS driver to seal the envelope
+    #[cfg(feature = "aliyun")]
     Ali(AliProviderArgs),
+
+    /// Intel eHSM driver to seal the envelope
+    #[cfg(feature = "ehsm")]
+    Ehsm(EhsmProviderArgs),
+
+    /// NoFeatureOpened is added in case no feature is opened, which is not prefered
+    NoFeatureOpened,
```

Comment:
It's a bit weird that this enum will still have `NoFeatureOpened` even if other features are enabled. I wonder if we want to create some kind of SampleKMS that can always be enabled as a fallback.

Reply:
KMS plugins may not be secure enough to save encrypt keys. If no KMS is supported, I think KBS is better to be the fallback to save secrets. And `NoFeatureOpened` is weird indeed. So I revised the implementation of this part. Hope it looks better.